src/i18n/en-US.json

```diff
@@ -295,7 +295,8 @@
   "accessibility.messageActionsMenuLike": "React with heart",
   "accessibility.messageActionsMenuEmoji": "Select emoji",
   "accessibility.messageActionsMenuLabel": "Message actions",
-  "conversationCreateReceiptsEnabled": "Read receipts are on",
+  "accessibility.emojiPickerSearchPlaceholder": "Search for emoji"
+,  "conversationCreateReceiptsEnabled": "Read receipts are on",
```

Comment:
Lets put this comma before the new line 😅

Reply:
ha ha, by mistake